app/data/census_household.json

```diff
@@ -2893,16 +3039,16 @@
                     "sections": [
                         {
                             "id": "disability-section",
```

Comment:
Should there be routing on this to skip to 27. Thinking of the last 12 months... if the person is under 16?